src/components/calcite-popover-manager/calcite-popover-manager.tsx

```diff
@@ -33,11 +46,25 @@ export class CalcitePopoverManager {
   //
   //--------------------------------------------------------------------------
 
-  @Listen("click", { capture: true }) toggle(event: Event) {
+  @Listen("click", { target: "window", capture: true }) closeOpenPopovers(
+    event: Event
+  ) {
     const target = event.target as HTMLElement;
-
+    const { autoClose, el, selector } = this;
+    const CALCITE_POPOVER_TAGNAME = "calcite-popover";
```

Comment:
Nitpick: Why do you need all upper casing here? It's a local var. 
Nitpick: Name reads like `Element#tagName` which is always uppercased.